content/en/data_jobs/kubernetes.md

```diff
@@ -65,6 +61,7 @@ You can install the Datadog Agent using the [Datadog Operator][3] or [Helm][4].
      global:
        tags:
          - 'data_workload_monitoring_trial:true'
+         - 'djm_config.enabled:true'
```

Comment:
@yiliangzhou @paul-laffon-dd  why are we adding this as a host tag instead of as an agent config setting? Don't we expect the agent config to be able to generate the metering metric?

Reply:
good catch. This is a mistake.